plugins/org.eclipse.papyrus.uml.interaction.model.edit/src-gen/org/eclipse/papyrus/uml/interaction/model/provider/MExecutionItemProvider.java

```diff
@@ -110,7 +110,7 @@ public Object getImage(Object object) {
 	@Override
 	public String getText(Object object) {
 		String label = ((MExecution) object).getName();
-		return (label == null) || (label.length() == 0) ? getString("_UI_MExecution_type") : //$NON-NLS-1$
```

Comment:
Again, not whitespace but formatting, which should not be changing with re-generation.

Reply:
Right, whitespace wasn't the right word. Formatting the src-gen folders didn't change anything (see previous comment).